### PR TITLE
Add minimal-web-notepad to Note-taking and Editors

### PR DESCRIPTION
**Pull request by domOrielton**
Add a fork of minimal-web-notepad that has additional basic note password protection, read only view and clickable links. Demo and source is all on GitHub page linked 

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the following requirements:
        
``- [Name](http://homepage/) - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Depends on proprietary services:

``- [Name](http://homepage/) `⚠` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Non-free software in `non-free.md`:

``- [Name](http://homepage/) `⊘ Proprietary` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``

- [ ] Your additions are ordered alphabetically.
- [ ] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md) and marked `⊘ Proprietary`.
- [ ] If your additions depend on proprietary services outside the user's control, they must be marked `⚠`.
- [ ] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [ ] Your additions have their `License` main server-side `Language`/platform/requirement listed.
- [ ] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [ ] You have searched the repository for any relevant [issues](https://github.com/Kickball/awesome-selfhosted/issues) or [PRs](https://github.com/Kickball/awesome-selfhosted/pulls), including closed ones.
- [ ] If the main link of your addition points to the program's source code, there is no need to add a duplicate _Source Code_ link. The _Demo_ and _Clients_ links are not mandatory.
- [ ] Any category you are creating has the minimum requirement of 3 items. If not, your addition may be inserted into `Misc/Other`.
- [ ] Any software project you are adding to the list is actively maintained.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1008,6 +1008,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
 - [Markdown Edit](http://georgeosddev.github.com/markdown-edit/) - Online markdown editor/viewer. ([Source Code](https://github.com/georgeOsdDev/markdown-edit)) `MIT` `HTML5`
 - [Meemo](https://meemo.minimal-space.de/) - Personal notes stream with Markdown support. ([Source Code](https://github.com/nebulade/meemo)) `MIT` `Nodejs`
 - [minimalist-web-notepad](https://github.com/pereorga/minimalist-web-notepad) - Minimalist notepad.cc clone. `Apache-2.0` `PHP`
+- [minimal-web-notepad](https://github.com/domOrielton/minimal-web-notepad) - Minimal online notepad `Apache-2.0` `PHP`
```
Comment (nodiscc) [inline README.md]: Missing final dot after description

**Comment by nodiscc** [comment]
@domOrielton have you considered forking https://github.com/pereorga/minimalist-web-notepad (the git repository), and submitting your patches there? The original is quite good (still maintained) and your modifications look good.

I tried to diff both projects but the changes are too large. It would be nice to have at least a common git history to know what was changed/how.

**Comment by domOrielton** [comment]
Interesting idea on the forking [ https://github.com/pereorga/minimalist-web-notepad] https://github.com/pereorga/minimalist-web-notepad and submitting the patches. This would take some time as there are quite a lot of additions to the original (which is why I renamed it from minimalist to minimal, not quite as tiny as the original), but if I need to do that to get it listed on the self-hosted list then I can get it done...

**Comment by nodiscc** [comment]
It is not absolutely required, but since you added sensitive functionality like password-protected notepads, and the history is quite [terse](https://github.com/domOrielton/minimal-web-notepad/commits/master), I would prefer that you start from a fork of the original, yes.

This way anyone can check that you did everything correctly security-wise, and it also makes it easier to merge back the features you added to the upstream project. You can submit patches latr but having a readable history is definitely good.

Please let us know if you intend to do that.

**Pull request closed without merging**